coops-ui/jest.config.js

```diff
@@ -13,9 +13,7 @@ module.exports = {
     '^.+\\.tsx?$': 'ts-jest',
     '^.+\\.(js|jsx)?$': 'babel-jest'
   },
-  transformIgnorePatterns: [
-    '/node_modules/(?!sbc-common-components/.*)'
-  ],
+  transformIgnorePatterns: [],
```

Comment:
Just clarifying, was this meant to also happen in this PR? As this change is also in code review in https://github.com/bcgov/lear/pull/630

Reply:
Sorry, I forgot to mention this. This was needed here for the unit tests to pass (even before I changed the unit tests). I assume it was broken previously. Since this PR will probably be merged before yours, you may have to remove this from yours.

Reply:
No problem at all! It shouldn't be an issue, it shouldn't even appear as a diff to whichever PR goes in second, i think.